tests/unit/test_discovery_client.py

```diff
@@ -183,6 +190,14 @@ def test___discovery_service_exe_unavailable___register_service___registration_f
     assert not registration_success_flag
 
 
+@pytest.fixture(scope="module")
+def subprocess_popen_creationflags() -> Dict[str, Any]:
```

Comment:
Rename to `subprocess_popen_kwargs`. `creationflags` is only one element of the kwargs and we will likely add a different one for Linux.